packages/lib/src/language/utils.ts

```diff
@@ -113,11 +113,12 @@ export const getTranslation = (translations: object, key: string, options: { [ke
 };
 
 /**
- * Returns an array with all the locales
+ * Returns an Object which contains all the key/values of the translation labels
+ *
  * @param locale - The locale the user wants to use
- * @param customTranslations -
+ * @param customTranslations - Custom translations provided by the merchant
  */
-export const loadTranslations = async (locale: string, customTranslations: object = {}) => {
+export const loadTranslations = async (locale: string, customTranslations: object = {}): Promise<Record<string, string>> => {
```

Comment:
Is this really the right return Type?

Reply:
Yes. It is asynchronous and it returns the merged `translations` object